### ISSUE TITLE
Allow multiple OR values for begins_with?

### ISSUE THREAD

**Issue by laggingreflex** [handsontable/handsontable#3660]
Currently `begins_with` can only be configured with a single value. I was wondering if it could be configured with multiple OR-like values, just like the `by_value` filter?

```
hot.getPlugin('filters').addFormula(0, 'begins_with', ['a', /*or*/ 'b']); 
```

I noticed that in `beginsWith.js` there's a variable specifically named `inputValues`

```
function formula(dataRow, [value] = inputValues) {
  return startsWith(stringify(dataRow.value).toLowerCase(), stringify(value));
}
```

...but only first value is actually being processed. Couldn't it be changed to something like:

```
function formula(dataRow, inputValues) {
  return inputValues.reduce((result, value) => result || startsWith(stringify(dataRow.value).toLowerCase(), stringify(value)), false);
}
```

**Comment by AMBudnik** [comment]
It's a good idea to add this feature. Especially if you can add many filters.

**Comment by AMBudnik** [comment]
Issue transfered to [wiki > feature request](https://github.com/handsontable/handsontable/wiki/Feature-requests)
It'll be reopened when ready to develop.

**Comment by krzysztofspilka** [comment]
@wszymanski What's the current status of this task?

**Comment by wszymanski** [comment]
When I will end #4262 I go back to this task. At first, I will use added feature. Then I will write tests for functionality added withing this task (API) and filters UI.

**Comment by swistach** [comment]
@wszymanski: check my comment in our internal issue board (GL#160)
We should remember about notify @laggingreflex, about new feature in filters, when we release new version of the Handsontable.

**Comment by wszymanski** [comment]
@laggingreflex, new Filters API was released within [release 1.12.0](http://docs.handsontable.com/pro/1.12.0/tutorial-release-notes.html). From now you can choose operations between conditions.

[**Disjunction**](https://en.wikipedia.org/wiki/Logical_disjunction) operation
```js
hot.getPlugin('filters').addCondition(1, 'begins_with', ['n'], 'disjunction');
hot.getPlugin('filters').addCondition(1, 'ends_with', ['l'], 'disjunction');
hot.getPlugin('filters').filter();
```

[**Conjunction**](https://en.wikipedia.org/wiki/Logical_conjunction) operation (by default)

```js
hot.getPlugin('filters').addCondition(1, 'begins_with', ['n'], 'conjunction');
hot.getPlugin('filters').addCondition(1, 'ends_with', ['l'], 'conjunction');
hot.getPlugin('filters').filter();
```

or just

```js
hot.getPlugin('filters').addCondition(1, 'begins_with', ['n']);
hot.getPlugin('filters').addCondition(1, 'ends_with', ['l']);
hot.getPlugin('filters').filter();
```

You can realize your proposal basing on below example:

```js
hot.getPlugin('filters').addCondition(0, 'begins_with', ['a'], 'disjunction');
hot.getPlugin('filters').addCondition(0, 'begins_with', ['b'], 'disjunction');
hot.getPlugin('filters').filter();
```
